pymtl3/passes/backends/yosys/test/TranslationImport_stdlib_test.py

```diff
@@ -10,17 +10,20 @@
 files and add my own test function (the `run_test`s). However we need
 to make sure the orignal reference is not lost and is restored after
 finishing each test (no matter it fails or passes).
+
+The following queue test cases are not ported to the Yosys backend
+because they don't use the test vector simulator:
+  test_normal1_simple,
```

Comment:
@ptpan Is it possible to have _run_queue_test for yosys backend similar to what you did above?

Reply:
Done. Maybe there's a better way to avoid duplication of `_run_queue_test`...